packages/chevrotain/src/parse/parser/traits/recoverable.ts

```diff
@@ -271,9 +275,12 @@ export class Recoverable {
     let nextToken = this.LA(1)
     let k = 2
     while (true) {
-      const nextTokenType: any = nextToken.tokenType
-      if (includes(allPossibleReSyncTokTypes, nextTokenType)) {
-        return nextTokenType
+      const foundMatch = allPossibleReSyncTokTypes.find((resyncTokType) => {
```

Comment:
We can use the lodash's `find` instead to remain consistent with the rest of the code base?

Reply:
Absolutely. I will fix it later.